docs/UserGuide.md

```diff
@@ -388,7 +402,7 @@ Example:
 
 #### Pinning a contact : `pin`
 
-Pins the specified contact on Pooch Planner so that your important contacts will consistently appear at the top on the contact list.
+Pins the specified contact on PoochPlanner so that your important contacts will consistently appear at the top on the contact list.
```

Comment:
change to "...at the top ~~on the~~ _of your_ contact list."

Reply:
changed